frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -1,9 +1,11 @@
 name: {{FRAMEWORK_NAME}}
 scheduler:
   principal: {{SERVICE_PRINCIPAL}}
+  user: {{TASKCFG_ALL_TASK_USER}}
```

Comment:
Ah alright, in that case I'll remove the wiring I setup in this PR to inject this into the FrameworkInfo protobuf then.